packages/main/test/specs/base/AriaLabelHelper.spec.js

```diff
@@ -0,0 +1,137 @@
+const assert = require('chai').assert;
+
+describe('AriaLabelHelper', () => {
+  before(async () => {
+    await browser.url(`test/pages/base/AriaLabelHelper.html`);
+  });
+
+  const getErrorMessageAriaLabelNotAsExpected = (actual, expected) => {
+    return `aria-label ${actual} is not as expected ${expected}.`;
+  };
+
+  const testInputAriaLabelMatchesLabels = async (inputId, labelIds) => {
+    const input = await browser.$(`#${inputId}`);
+    const innerInput = await input.shadow$('input');
+    const actualAriaLabel = await innerInput.getAttribute('aria-label');
+
+    const promises = labelIds.map(async (labelId) => {
+      const label = await browser.$(`#${labelId}`);
+      return await label.getText();
+    });
+    const texts = await Promise.all(promises);
+    const expectedAriaLabel = texts.join(' ');
+
+    assert.equal(
+      actualAriaLabel,
+      expectedAriaLabel,
+      getErrorMessageAriaLabelNotAsExpected(actualAriaLabel, expectedAriaLabel)
+    );
+  };
+
+  const testInputAriaLabelMatchesAccessibleName = async (inputId) => {
+    const input = await browser.$(`#${inputId}`);
+    const innerInput = await input.shadow$('input');
+    const accessibleNameValue = await input.getAttribute('accessible-name');
+    const actualAriaLabel = await innerInput.getAttribute('aria-label');
+    assert.equal(
+      actualAriaLabel,
+      accessibleNameValue,
+      getErrorMessageAriaLabelNotAsExpected(actualAriaLabel, accessibleNameValue)
```

Comment:
Don't we need to return a positive message here not getErrorMessageAriaLabelNotAsExpected?

Reply:
I thought the same, but since the message is only shown when there is an error, I was ok to leave it like this.

I checked the docs for `chai` and the example was:

```js
assert.strictEqual(true, true, 'these booleans are strictly equal');
```

so the message is indeed expected to be positive. I don't have a strong opinion, but yes, it would be best to make all messages positive to be consistent with the other components' tests.